qiita_pet/static/vendor/js/resumable-uploader.js

```diff
@@ -50,10 +50,10 @@
        this.resumable = new Resumable({
            chunkSize:3*1024*1024,
            maxFileSize:this.maxFileSize*1024*1024*1024,
-           simultaneousUploads: 6,
+           simultaneousUploads: 1,
```

Comment:
In the past we were saving all chunks which allowed us to do parallel uploads. Now, we save everything in the same file and we need the chucks in a defined order, thus 1 by 1.